ci/test_docker.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+# Test Snowflake SQLAlchemy in Docker
+# NOTES:
+#   - By default this script runs Python 3.7 tests, as these are installed in dev vms
+#   - To compile only a specific version(s) pass in versions like: `./test_docker.sh "3.7 3.8"`
+
+set -o pipefail
+
+# In case this is ran from dev-vm
+PYTHON_ENV=${1:-3.7}
+
+# Set constants
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SQLALCHEMY_DIR="$( dirname "${THIS_DIR}")"
+WORKSPACE=${WORKSPACE:-$SQLALCHEMY_DIR}
+source $THIS_DIR/set_base_image.sh
+
+cd $THIS_DIR/docker/sqlalchemy_build
+
+CONTAINER_NAME=test_sqlalchemy
+
+echo "[Info] Building docker image"
+arch=$(uname -p)
+
+echo "[Info] Building docker image"
+if [[ "$arch" == "aarch64" ]]; then
+  BASE_IMAGE=$BASE_IMAGE_MANYLINUX2014AARCH64
+  GOSU_URL=https://github.com/tianon/gosu/releases/download/1.14/gosu-arm64
+else
+  BASE_IMAGE=$BASE_IMAGE_MANYLINUX2014
+  GOSU_URL=https://github.com/tianon/gosu/releases/download/1.14/gosu-amd64
+fi
```

Comment:
```suggestion
GOSU_URL=https://github.com/tianon/gosu/releases/download/1.14/gosu-arm64
if [[ "$arch" == "aarch64" ]]; then
  BASE_IMAGE=$BASE_IMAGE_MANYLINUX2014AARCH64
else
  BASE_IMAGE=$BASE_IMAGE_MANYLINUX2014
fi
```

Reply:
qq: this part is copied from connector, the gosu release seems to be platform dependent -- arm vs amd?

Reply:
Ahh, I missed that last part of the URLs